pulsar-broker/src/test/java/org/apache/pulsar/common/naming/NamespaceBundlesTest.java

```diff
@@ -89,8 +91,9 @@ public void testConstructor() throws Exception {
 
     @SuppressWarnings("unchecked")
     private NamespaceBundleFactory getNamespaceBundleFactory() {
-        PulsarService pulsar = mock(PulsarService.class);
+        PulsarService pulsar = mock(PulsarService.class); 
```

Comment:
Remove the blank in the tail

```suggestion
        PulsarService pulsar = mock(PulsarService.class);
```